airtime_mvc/application/models/Block.php

```diff
@@ -1758,6 +1749,8 @@ public function getListofFilesMeetCriteria($showLimit = null)
         // check if file exists
         $qry->add("file_exists", "true", Criteria::EQUAL);
         $qry->add("hidden", "false", Criteria::EQUAL);
+
+        //$qry->addAsColumn('lptime','COALESCE("lptime",\'1970-01-01\')');
```

Comment:
Does this comment serve a purpose?

Reply:
Nope it was a failed attempt at accomplishing what I ended up doing in the last commit. I spent many hours digging into Propel attempting to figure out the sort only to realize that it could be done rather easily by modifying the sort statement to sort nulls first with parenthesis. Funny how walking away from a problem and coming at it fresh can sometimes result in instant fixes. I'll push a quick commit to remove this.